shuup/admin/modules/orders/views/list.py

```diff
@@ -60,7 +60,7 @@ class OrderListView(PicotableListView):
         Column("payment_status", _(u"Payment Status"), filter_config=ChoicesFilter(choices=PaymentStatus.choices)),
         Column("shipping_status", _(u"Shipping Status"), filter_config=ChoicesFilter(choices=ShippingStatus.choices)),
         Column(
-            "taxful_total_price", _(u"Total"), sort_field="taxful_total_price_value",
+            "taxful_total_price_value", _(u"Total"), sort_field="taxful_total_price_value",
```

Comment:
Not sure if it's because of this change but unselecting total price causes KeyError at /sa/orders 'taxful_total_price_value'